File: 1468-check-if-n-and-its-double-exist/check-if-n-and-its-double-exist.py
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        for i in range(len(arr)):
            j = 0
            while j<len(arr):
                if arr[i]==2*arr[j] and i!=j:
                    return True
                j+=1

        return False

File: 1468-check-if-n-and-its-double-exist/check-if-n-and-its-double-exist.cpp
class Solution {
public:
    bool checkIfExist(vector<int>& arr) {
        int i, j;
        for(int i=0; i<arr.size(); i++){
            j = 0;
            while(j<arr.size()){
                if(arr[i]==2*arr[j] && i!=j){
                    return true;
                }
                j++;
            }
        }
        return false;
    }
};